www/include/Administration/parameters/DB-Func.php

```diff
@@ -970,9 +989,32 @@ function updateKnowledgeBaseData($db, $form, $centreon)
 
     foreach ($ret as $key => $value) {
         if (preg_match('/^kb_/', $key)) {
-                updateOption($db, $key, $value);
+            updateOption($db, $key, $value);
         }
     }
 
     $centreon->initOptGen($db);
 }
+
+/**
+ * Used to update Central's credentials on a remote server
+ *
+ * @param $db : database connection
+ * @param $form : form data
+ */
+function updateRemoteAccessCredentials($db, $form)
+{
+    $ret = $form->getSubmitValues();
+    updateInformations(
```

Comment:
security alert: the password is stored in clear text in the database

Reply:
I guess it is what we said during last meeting.
This is the first implementation, and we'll use an other way at next step

Reply:
@callapa , As discussed, I added encryption